invenio_records_resources/resources/files/resource.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, config, service):
     def create_url_rules(self):
         """Routing for the views."""
         routes = self.config.routes
+
```

Comment:
minor: does this fix black/lint issues? or can it be avoided?

Reply:
I think it can be avoided, want me to edit it?